packages/webapp/src/routes/credits-purchase/check-defra-account-details.js

```diff
@@ -0,0 +1,38 @@
+import creditsPurchaseConstants from '../../utils/credits-purchase-constants.js'
+import getApplicantContext from '../../utils/get-applicant-context.js'
+
+const handlers = {
+  get: async (request, h) => {
+    // Clear any previous confirmation every time this page is accessed as part of forcing the user to confirm
+    // their account details are correct based on who they are representing in the current session.
+    request.yar.get(creditsPurchaseConstants.redisKeys.DEFRA_ACCOUNT_DETAILS_CONFIRMED, true)
+    return h.view(creditsPurchaseConstants.views.CREDITS_PURCHASE_CHECK_DEFRA_ACCOUNT_DETAILS, getApplicantContext(request.auth.credentials.account, request.yar))
+  },
+  post: async (request, h) => {
+    const defraAccountDetailsConfirmed = request.payload.defraAccountDetailsConfirmed
+    if (defraAccountDetailsConfirmed) {
+      request.yar.set(creditsPurchaseConstants.redisKeys.DEFRA_ACCOUNT_DETAILS_CONFIRMED, defraAccountDetailsConfirmed)
+      // TODO: Update redirection to task list when task list task is complete
+      // return h.redirect(creditsPurchaseConstants.routes.REGISTER_CREDIT_PURCHASE_TASK_LIST)
+      return h.redirect('#')
```

Comment:
Please add minimal route, view and unit test support for the credits purchase task list rather than redirecting back to the same page. This pattern allows another developer to replace the minimal implementation in a subsequent story and does not require commented code. There are lots of examples of this in https://github.com/DEFRA/biodiversity-net-gain-service/pull/515

Reply:
.No longer required in this pull request due to inclusion in https://github.com/DEFRA/biodiversity-net-gain-service/pull/615